### PR TITLE
Added normalize.css in all the html files and a navigation menu

### PR DESCRIPTION
**Pull request by sahibkaur**

### CODE REVIEW

File: css/styles.css [hunk 1]
```diff
@@ -2,6 +2,21 @@ body {
...
+    justify-content: space-evenly;
+    height: 500px;
+}
+
+.boxes {
+    width: 100px;
+    height: 100px;
+    background-color: antiquewhite;
+    text-align: center;
+
```
Comment (theamanjs) [inline css/styles.css]: Shouldn't have empty line here

File: index.html [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/styles.css" />
+</head>
+<body>
+    <div class="container">
+        <div class="boxes"><a href="part1.html">Part 1</a></div>
+        <div class="boxes"><a href="part2.html">Part 2</a></div>
+    </div>
+</body>
+</html>
```
Comment (theamanjs) [inline index.html]: Should have new line at the end of the file

File: part1.html [hunk 3]
```diff
@@ -41,7 +41,7 @@ <h2> Part 1</h2>
             </tr>
        </table>
        <hr>
-       <h2> Part 2</h2>
+       <h2> Exercise 2</h2>
        <div class="red-box"></div>
     </body>
 </html>
```
Comment (theamanjs) [inline part1.html]: Should have new line at the end of the file

File: part2.html [hunk 4]
```diff
@@ -5,6 +5,7 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Document</title>
+    <link rel="stylesheet" href="css/normalize.css" />
```
Comment (theamanjs) [inline part2.html:8]: Are you aware of what the normalize CSS does.

**Review by theamanjs — CHANGES_REQUESTED** [review]

**Comment by theamanjs** [comment]
For good practices, PRs should have a description of the things that are done.

**Comment by Gauravjeetsingh** [comment]
LGTM 👍

**Review by Gauravjeetsingh — APPROVED** [review]

**Comment by sahibkaur** [comment]
> For good practices, PRs should have a description of the things that are done.

Fixed

**Review by theamanjs — APPROVED** [review]

**Pull request merged**